doc/DEVELOPER_TOOLING.md

```diff
@@ -26,7 +26,7 @@ make astyle
 If you have only `astyle` then use:
 
 ```BASH
-astyle --options=.astylerc --recursive src/*.cpp,*.h tests/*.cpp,*.h`
+astyle --options=.astylerc --recursive src/*.cpp,*.h tests/*.cpp,*.h --exclude=src/third-party
```

Comment:
I suggest putting the new option into `.astylerc` instead.

Also, it seems `--ignore-exclude-errors` is also needed because otherwise astyle freaks out over `tests/*.cpp` not containing `src/third-party`.

Reply:
Thanks for pointing that out! Incorporated this feedback.